src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.figures.figure.Circle;
+import core.basesyntax.figures.figure.IsoscelesTrapezoid;
+import core.basesyntax.figures.figure.Rectangle;
+import core.basesyntax.figures.figure.RightTriangle;
+import core.basesyntax.figures.figure.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    private static final String COLOR = "WHITE";
+    private static final int RADIUS = 10;
+    private Random random;
+    private final ColorSupplier colorSupplier;
+
+    public FigureSupplier(Random random, ColorSupplier colorSupplier) {
+        this.random = random;
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Figure getRandomFigure() {
+        String[] figures = {"circle", "isoscelesTrapezoid", "rectangle", "rightTriangle", "square"};

```

Comment:
create a separate enum for storing such values and use it with switch case instead of if else